app/src/main/java/com/futurice/freesound/feature/search/SearchFragmentViewModel.java

```diff
@@ -74,9 +75,8 @@ void openSoundDetails(@NonNull final Sound sound) {
 
     @NonNull
     private static List<DisplayableItem> wrapInDisplayableItem(@NonNull final List<Sound> sounds) {
-        return Observable.fromIterable(sounds)
-                         .map(sound -> DisplayableItem.create(sound, SOUND))
-                         .toList()
-                         .blockingGet();
+        return Ix.from(sounds)
```

Comment:
It may become unnecessary once Java 8 streams are used, but it's generally faster than RxJava2 for synchronous stuff.